apps/addons/tests/test_models.py

```diff
@@ -2390,29 +2391,57 @@ def create_addon(self, **kwargs):
         addon.save()
         return addon
 
-    def test_increment_udpated_status(self):
+    def addon_attr(self, **overrides):
+        return self.create_addon(**overrides).__dict__.copy()
+
+    def test_increment_updated_status(self):
         addon = self.create_addon()
-        with patch('addons.models.statsd.incr') as mock_incr:
+        with patch('addons.models.track_addon_status_change') as mock_:
             addon.update(status=amo.STATUS_PUBLIC)
-        mock_incr.assert_called_with(
-            'addon_status_change.status_{}'.format(amo.STATUS_PUBLIC)
-        )
+
+        d = addon.__dict__.copy()
+        d['status'] = amo.STATUS_PUBLIC
+        assert (sorted(mock_.call_args[0][0].keys()) ==
+                sorted(k for k in d.keys() if not k.startswith('__')))
```

Comment:
Feels like these tests are fighting the fact that a `dict` is being passed in. Even though it isn't great if you can't use an instance I'd suggest passing two keyword arguments to it instead.